### PR TITLE
[zh]Sync reference-12

### PR DESCRIPTION
**Pull request by nameYULI**
related issue: https://github.com/kubernetes/website/issues/33478
reference-12
```
56	22	content/en/docs/reference/kubectl/cheatsheet.md
22	22	content/en/docs/reference/access-authn-authz/admission-controllers.md
```
content/zh/docs/reference/kubectl/conventions.md is still in sync

### CODE REVIEW

File: content/zh/docs/reference/access-authn-authz/admission-controllers.md [hunk 1]
```diff
@@ -932,16 +897,15 @@ Such kubelets will only be allowed to modify their own `Node` API object, and on
...
 <!--
-In Kubernetes 1.11+, kubelets are not allowed to update or remove taints from their `Node` API object.
+kubelets are not allowed to update or remove taints from their `Node` API object.
 
-In Kubernetes 1.13+, the `NodeRestriction` admission plugin prevents kubelets from deleting their `Node` API object,
+The `NodeRestriction` admission plugin prevents kubelets from deleting their `Node` API object,
 and enforces kubelet modification of labels under the `kubernetes.io/` or `k8s.io/` prefixes as follows:
 -->
-在 Kubernetes 1.11+ 的版本中，不允许 kubelet 从 `Node` API 对象中更新或删除污点。
+不允许 kubelet 从 `Node` API 对象中更新或删除污点。
```
Comment (tengqm) [inline content/zh/docs/reference/access-authn-authz/admission-controllers.md:905]:
```suggestion
不允许 kubelet 在 `Node` API 对象上更新或删除污点。
```

File: content/zh/docs/reference/access-authn-authz/admission-controllers.md [hunk 2]
```diff
@@ -1283,21 +1222,26 @@ objects in your Kubernetes deployment, you MUST use this admission controller to
...
 
 <!--
-See the [resourceQuota design doc](https://git.k8s.io/community/contributors/design-proposals/resource-management/admission_control_resource_quota.md) and the [example of Resource Quota](/docs/concepts/policy/resource-quotas/) for more details.
+See the [ResourceQuota API reference](/docs/reference/kubernetes-api/policy-resources/resource-quota-v1/)
+and the [example of Resource Quota](/docs/concepts/policy/resource-quotas/) for more details.
 -->
 请查看
-[resourceQuota 设计文档](https://git.k8s.io/community/contributors/design-proposals/admission_control_resource_quota.md)和 [Resource Quota 例子](/zh/docs/concepts/policy/resource-quotas/)
-了解更多细节。
+[resourceQuota 设计文档](/zh/docs/reference/kubernetes-api/policy-resources/resource-quota-v1/)
```
Comment (tengqm) [inline content/zh/docs/reference/access-authn-authz/admission-controllers.md:1229]:
```suggestion
[resourceQuota API 参考](/zh/docs/reference/kubernetes-api/policy-resources/resource-quota-v1/)
```

File: content/zh/docs/reference/kubectl/cheatsheet.md [hunk 3]
```diff
@@ -648,6 +682,44 @@ kubectl top pod POD_NAME --containers               # 显示给定 Pod 和其中
...
+<!--
+```bash
+kubectl cp /tmp/foo_dir my-pod:/tmp/bar_dir            # Copy /tmp/foo_dir local directory to /tmp/bar_dir in a remote pod in the current namespace
+kubectl cp /tmp/foo my-pod:/tmp/bar -c my-container    # Copy /tmp/foo local file to /tmp/bar in a remote pod in a specific container
+kubectl cp /tmp/foo my-namespace/my-pod:/tmp/bar       # Copy /tmp/foo local file to /tmp/bar in a remote pod in namespace my-namespace
+kubectl cp my-namespace/my-pod:/tmp/foo /tmp/bar       # Copy /tmp/foo from a remote pod to /tmp/bar locally
+```
+-->
+```bash
+kubectl cp /tmp/foo_dir my-pod:/tmp/bar_dir            # 将 /tmp/foo_dir 本地目录复制到当前命名空间中远程 pod 中的 /tmp/bar_dir
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/cheatsheet.md:699]:
```suggestion
kubectl cp /tmp/foo_dir my-pod:/tmp/bar_dir            # 将 /tmp/foo_dir 本地目录复制到远程当前命名空间中 Pod 中的 /tmp/bar_dir
```

File: content/zh/docs/reference/kubectl/cheatsheet.md [hunk 4]
```diff
@@ -648,6 +682,44 @@ kubectl top pod POD_NAME --containers               # 显示给定 Pod 和其中
...
+kubectl cp /tmp/foo my-pod:/tmp/bar -c my-container    # 将 /tmp/foo 本地文件复制到特定容器中远程 pod 中的 /tmp/bar
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/cheatsheet.md:700]:
```suggestion
kubectl cp /tmp/foo my-pod:/tmp/bar -c my-container    # 将 /tmp/foo 本地文件复制到远程 Pod 中特定容器的 /tmp/bar 下
```

File: content/zh/docs/reference/kubectl/cheatsheet.md [hunk 5]
```diff
@@ -648,6 +682,44 @@ kubectl top pod POD_NAME --containers               # 显示给定 Pod 和其中
...
+kubectl cp /tmp/foo my-namespace/my-pod:/tmp/bar       # 将 /tmp/foo 本地文件复制到远程 “my-namespace” 命名空间中 pod 中的 /tmp/bar
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/cheatsheet.md:701]:
```suggestion
kubectl cp /tmp/foo my-namespace/my-pod:/tmp/bar       # 将 /tmp/foo 本地文件复制到远程 “my-namespace” 命名空间内指定 Pod 中的 /tmp/bar
```

File: content/zh/docs/reference/kubectl/cheatsheet.md [hunk 6]
```diff
@@ -648,6 +682,44 @@ kubectl top pod POD_NAME --containers               # 显示给定 Pod 和其中
...
+kubectl cp my-namespace/my-pod:/tmp/foo /tmp/bar       # 将 /tmp/foo 从远程 pod 复制到本地 /tmp/bar
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/cheatsheet.md:702]:
```suggestion
kubectl cp my-namespace/my-pod:/tmp/foo /tmp/bar       # 将 /tmp/foo 从远程 Pod 复制到本地 /tmp/bar
```

File: content/zh/docs/reference/kubectl/cheatsheet.md [hunk 7]
```diff
@@ -648,6 +682,44 @@ kubectl top pod POD_NAME --containers               # 显示给定 Pod 和其中
...
+```
+
+<!--
+`kubectl cp` requires that the 'tar' binary is present in your container image. If 'tar' is not present,`kubectl cp` will fail.
+For advanced use cases, such as symlinks, wildcard expansion or file mode preservation consider using `kubectl exec`.
+-->
+{{< note >}}
+`kubectl cp` 要求容器镜像中存在 'tar' 二进制文件。 如果 'tar' 不存在，`kubectl cp` 将失败。
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/cheatsheet.md:710]:
```suggestion
`kubectl cp` 要求容器镜像中存在 'tar' 二进制文件。如果 'tar' 不存在，`kubectl cp` 将失败。
```

File: content/zh/docs/reference/kubectl/cheatsheet.md [hunk 8]
```diff
@@ -857,15 +929,15 @@ Verbosity | Description
...
+* Read the [kubectl overview](/docs/reference/kubectl/) and learn about [JsonPath](/docs/reference/kubectl/jsonpath).
 
 * See [kubectl](/docs/reference/kubectl/kubectl/) options.
 
 * Also read [kubectl Usage Conventions](/docs/reference/kubectl/conventions/) to understand how to use kubectl in reusable scripts.
 
 * See more community [kubectl cheatsheets](https://github.com/dennyzhang/cheatsheet-kubernetes-A4).
 -->
-* 参阅 [kubectl 概述](/zh/docs/reference/kubectl/overview/)，进一步了解[JsonPath](/zh/docs/reference/kubectl/jsonpath)。
+* 参阅 [kubectl 概述](/zh/docs/reference/kubectl/)，进一步了解[JsonPath](/zh/docs/reference/kubectl/jsonpath)。
```
Comment (tengqm) [inline content/zh/docs/reference/kubectl/cheatsheet.md:940]:
```suggestion
* 参阅 [kubectl 概述](/zh/docs/reference/kubectl/)，进一步了解 [JsonPath](/zh/docs/reference/kubectl/jsonpath)。
```

File: content/zh/docs/reference/kubectl/cheatsheet.md [hunk 9]
```diff
@@ -70,6 +72,18 @@ source <(kubectl completion zsh)  # 在 zsh 中设置当前 shell 的自动补
...
+-->
+### 关于 --all-namespaces 的一点说明
+
+<!--
+Appending `--all-namespaces` happens frequently enough where you should be aware of the shorthand for `--all-namespaces`:
+-->
+附加 `--all-namespaces` 经常发生，你应该知道 `--all-namespaces` 的简写：
+
+```kubectl -A```
+
```
Comment (Sea-n) [inline content/zh/docs/reference/kubectl/cheatsheet.md:86]: How about this wording:

```
我們經常用到 `--all-namespaces` 參數，你應該要知道以下簡寫：
```

File: content/zh/docs/reference/kubectl/cheatsheet.md [hunk 10]
```diff
@@ -140,6 +158,11 @@ kubectl config set-context gce --user=cluster-admin --namespace=foo \
   && kubectl config use-context gce
 
 kubectl config unset users.foo                       # 删除用户 foo
+
+# 设置/显示上下文/命名空间的短别名（仅适用于 bash 和 bash 兼容的 shell
+# 在使用 kn 设置命名空间之前要设置当前上下文）
```
Comment (Sea-n) [inline content/zh/docs/reference/kubectl/cheatsheet.md:163]:
```suggestion
# 設置或顯示 context / namespace 的短别名
# （仅适用于 bash 和兼容 bash 的 shell，在使用 kn 设置命名空间之前要先设置 current-context）
```

File: content/zh/docs/reference/kubectl/cheatsheet.md [hunk 11]
```diff
@@ -648,6 +682,44 @@ kubectl top pod POD_NAME --containers               # 显示给定 Pod 和其中
...
+```
+
+<!--
+`kubectl cp` requires that the 'tar' binary is present in your container image. If 'tar' is not present,`kubectl cp` will fail.
+For advanced use cases, such as symlinks, wildcard expansion or file mode preservation consider using `kubectl exec`.
+-->
+{{< note >}}
+`kubectl cp` 要求容器镜像中存在 'tar' 二进制文件。如果 'tar' 不存在，`kubectl cp` 将失败。
+对于高级用例，例如符号链接、通配符扩展或文件模式保存，请考虑使用 `kubectl exec`。
+{{< /note >}}
```
Comment (Sea-n) [inline content/zh/docs/reference/kubectl/cheatsheet.md:712]:
```suggestion
<!--
`kubectl cp` requires that the 'tar' binary is present in your container image. If 'tar' is not present,`kubectl cp` will fail.
For advanced use cases, such as symlinks, wildcard expansion or file mode preservation consider using `kubectl exec`.
-->
{{< note >}}
`kubectl cp` 要求容器镜像中存在 “tar” 二进制文件。如果 “tar” 不存在，`kubectl cp` 将失败。
对于進階用例，例如符号链接、通配符扩展或保留文件權限，请考虑使用 `kubectl exec`。
{{< /note >}}
```
